api/models/user.py

```diff
@@ -15,14 +15,25 @@ class Submission(Document):
     identifier: PydanticObjectId
     submitted: datetime = datetime.utcnow()
     url: HttpUrl = None
-    repository: Optional[str]
-    repository_identifier: Optional[str]
+    repository: Optional[str] = None
+    repository_identifier: Optional[str] = None
+
+    @model_validator(mode='after')
+    def url_to_string(self):
+        if self.url is not None:
+            self.url = str(self.url)
```

Comment:
I'm missing something on these httpurl to str conversions.  Why is this change needed?